model_compression_toolkit/exporter/model_exporter/keras/fakely_quant_keras_exporter.py

```diff
@@ -115,6 +119,18 @@ def _unwrap_quantize_wrapper(layer: Layer):
                                                           input_tensors=None,
                                                           clone_function=_unwrap_quantize_wrapper)
 
+        # Transform the model's configuration to convert KerasActivationQuantizationHolder to fake-quant layers
+        new_cfg = self.transform_model_cfg()
+
+        # Create a new model with the transformed configuration
+        new_model = tf.keras.Model().from_config(new_cfg)
+
+        # Set the filtered weights (weights of the original model without optimization steps variables)
+        # to the new model
+        filtered_weights = self.get_filtered_weights()
```

Comment:
filter "optimization step" variables from the weights of the model (they are only for training )